Automattic/jetpack
# Fix XSS vulnerability in js packages

Addresses https://github.com/Automattic/jetpack/security/dependabot/87, https://github.com/Automattic/jetpack/security/dependabot/86.

## Proposed changes:

* Update rollup to a newer version. We should switch to webpack (https://github.com/Automattic/jetpack/issues/38561).

### Other information:


## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

no

## Testing instructions:

### Test that building works

- Run `pnpm build` for both packages;
- There should be no issues building.

### Test that Image Guide works

- Make sure the package is built using `jetpack build js-packages/image-guide`;
- Run `jetpack watch plugins/boost` to update the image-guide JS in the plugin;
- Enable Image Guide from the Boost UI;
- Check a page with images and make sure there are no errors related to Image Guide.
